packages/qdrant-loader/src/qdrant_loader/core/worker/pool.py

```
@@ -0,0 +1,109 @@
+
+    DEFAULT_SOURCE_KEY = "__global__"
+
+    def __init__(
+        self,
+        queue: JobQueue,
+        handler: JobHandler,
+        worker_count: int = 4,
+        lease_seconds: int = 60,
+    ) -> None:
+        if worker_count < 1:
+            raise ValueError("worker_count must be >= 1")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)